.gitignore

```diff
@@ -10,3 +10,5 @@ extension.css
 report.html
 stats.json
 analyze.txt
+*.woff2
+*.woff
```

Comment:
heads up as this will be annoying for a bit. Tldraw generates a bunch of `.woff` files, and because we copy everything in `dist` to `root` bc that's where Roam expects it, it will clutter your top level folder